signer/keydbstore/sql_keydbstore_postgres.go

```diff
@@ -0,0 +1,24 @@
+// +build postgresqldb
```

Comment:
Is the model should be the same no matter what type of GORM DB it is, right?  (it might not be - Postgres may have different type declarations, but right now I don't see any difference between the two files) If so, should it then remain in `sql_keydbstore.go`?  The intention was for that to be the common code for all GORM related stuff.

I think maybe we can then convert all the `mysqldb` buildtags to just `sqldb` build tags, because the logic is the same (poll DB until we can contact it) and use it for both postgres and mysql - we just pass different DB url values to it.